universal-login-sdk/test/core/services/WalletSerializer.test.ts

```diff
@@ -104,7 +104,7 @@ describe('UNIT: WalletSerializer', () => {
         wallet: TEST_DEPLOYED_WALLET,
       })).to.deep.eq({
         kind: 'Deployed',
-        wallet: TEST_APPLICATION_WALLET,
+        wallet: {...TEST_APPLICATION_WALLET, email: undefined},
```

Comment:
I guess it is unnecessary

Reply:
Here is necessary, because {email: undefined}  and {} are not equal